### ISSUE TITLE
Release 4.6

### ISSUE THREAD

**Issue by phpgeekfr** [blockframes/blockframes#9151]
Issues:

Change-log: 
TODO

Tests : 
TODO

**Comment by 2amarkina** [comment]
### Archipel Content
**Interaction Buttons**
- [ ] In the Seller's notification, when a Buyer sends a Screener request, is it possible  to use the full name of activity as in this dropbox ? 

from 
![Screener Request Notification.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/d5346fa4-a038-4475-ae55-4b356dc9e56e)
to
![Screener Request Notification2.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/5440d416-b90a-409a-a87d-df16031bea61)

- [ ] In the same notification country should start with a capital letter
![Capture d’écran 2023-01-26 à 11.27.45.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/7006918d-67e8-4e3f-9913-32262d3b187e)

**Comment by 2amarkina** [comment]
### Archipel Content
**Modal "Create Offer" emails and notifications**

- [ ] As we removed "Delivery Specifications field, we need to remove it also from the emails we send to Buyers and Sellers 
Buyers
![Capture d’écran 2023-01-26 à 14.19.50.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/a2445dcd-e37a-4798-a4e9-09e1f7d95e53)

Sellers
![Capture d’écran 2023-01-26 à 14.19.43.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/a2f0e3a1-3b98-4a2f-8d20-4bc78b388d01)

**Comment by phpgeekfr** [comment]
>     * [ ]  As we removed "Delivery Specifications field, we need to remove it also from the emails we send to Buyers and Sellers
>       Buyers

Yes, this is part of the actions to perform right after release CF https://www.notion.so/cascade8/Release-troubleshooting-8085a54303de4019b0a129116d60fa13

**Comment by 2amarkina** [comment]
### Archipel Content
**Screener Request**

- [ ] If possible, change "Continue Browsing" by "Upload Screener". This link would lead a Seller to the Movie Tunnel, /media-screener page of the concerned movie
![Capture d’écran 2023-01-26 à 14.26.17.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/daf7d7e2-1fdd-4133-95e8-b14ed7eb9301)

**Comment by 2amarkina** [comment]
### General Bug
- [ ] The title "Storyline Elements" should be hidden if there is no information inside 
![Storyline elements block.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/8aecc4a2-ab92-490f-9c7e-050c7f8700c5)

### Semi-Public Access
- [ ] If a Seller has no titles published (only drafts or no titles at all) we should not show his line-up page (no matter anonymous or connected user)
![Showing Line-up with no Titles.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/ebac98d2-623d-4130-b1ff-e78a7317b2a0)

**Comment by 2amarkina** [comment]
### Semi-Public Access
**Analytics Data**

- [ ] When connecting as anonymous and then creating an org, the organization name is --deleted org-- . Might be related to the fact that the concerned email address has been already used for the previous tests.
![Capture d’écran 2023-01-27 à 12.10.17.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/ca0ccb00-ceab-4a5b-9bb8-6ca560c965a2)

**Comment by 2amarkina** [comment]
### Archipel Market
**One Particular Title Statistic Page**

**! _bonus, if it doesn't take too much time_ !**
At the bottom of the page we've got a block Screening performance. For the moment, if there is no screenings for the film, we show this section like shown on screenshot 
<img width="1152" alt="Снимок экрана 2023-01-30 в 10 55 43" src="https://user-images.githubusercontent.com/55499036/215446021-7abcd3e7-5cdf-4591-a768-dff8111942ed.png">

Could we introduce an empty state for this part as we do for the Interactions and Film Page views as shown on a mock-up below? 
<img width="658" alt="Снимок экрана 2023-01-30 в 11 01 23" src="https://user-images.githubusercontent.com/55499036/215446807-cda61678-abb5-411f-aae7-022b3b92b62a.png">

P.S. keep the actual wording (_Screening performance_ and _Check the cumulative statistics of this title's Screenings._)

If too much time needed, I'll transform this in future nice-to-have issue. 
@Jackseed what do you think?

**Comment by phpgeekfr** [comment]
> At the bottom of the page we've got a block Screening performance. For the moment, if there is no screenings for the film, we show this section like shown on screenshot 

The mockup is already implemented and I was not able to reproduce the issue. Maybe a slow network issue.. To prevent this I added a loader for this block that will be displayed until screening cards data is loaded

@2amarkina some remarks :

- [ ] if a movie is in draft I can click on "organize screening" from analytics tab but I cannot choose the movie from the list since it only shows accepted movies
- [ ] If I create a screening from this same button, minute for "start" and "end" can be like 14h06 but we can choose time only with a 30 min interval
 ![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/a9a4584b-b3a7-403e-82e0-e105ce3ba18b)

**Comment by 2amarkina** [comment]
> if a movie is in draft I can click on "organize screening" from analytics tab but I cannot choose the movie from the list since it only shows accepted movies

If a movie is in draft, then normally Seller will not have an access to the Titles Activity Rapport page a the data from the Buyers can appear only with the published Titles. So it should be a problem. 

> If I create a screening from this same button, minute for "start" and "end" can be like 14h06 but we can choose time only with a 30 min interval
For me it's not a problem as a user can still enter the needed time using keyboard.

**Comment by phpgeekfr** [comment]
- [x] If time from date picker is removed (modal or event page) , page crash when saving. Solved by adding a default `00:00`
![image.png](https://images.zenhubusercontent.com/5c6e955fdecd2e3c69b6c1d2/ae3e6e45-3836-43ea-ae5d-74be1f68ad1d)

**Comment by 2amarkina** [comment]
One last modification for this release, if possible: 

When a Seller has no titles in the line-up we need to put a placeholder to prevent a Buyer that it's not a bug of the platform. 

- [ ] We need to add, using the following structure, the text : 
Title: `The Seller has not added the Titles into the digital catalogue yet.`
Subtitle: `Come back later to check it out`
[Image](https://www.figma.com/file/Yi5RKQuroGJtpnPUCcAO67/Assets?node-id=520%3A286&t=wFyfgelqZb6ltqms-4)

Support Screenshot 
![Capture d’écran 2023-02-01 à 12.43.05.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/9539a622-a159-4484-a1be-e02ec63d47de)

**Comment by phpgeekfr** [comment]
- [ ] added placeholder for org line-up and a loader
- [ ] updated mailto email & subject when request is made from catalog app (should remain the same for festival)
